curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md

```diff
@@ -36,10 +36,17 @@ assert(
 The link's text should be `cat photos`. You have either omitted the text or have a typo.
 
 ```js
-const nestedAnchor = document.querySelector(`p > a`);
-assert(
-  nestedAnchor.innerText.toLowerCase().replace(/\s+/g, ' ') === 'cat photos'
-);
+const nestedAnchor = document.querySelector('p > a');
+const innerContent = nestedAnchor.innerHTML;
+assert.isTrue(innerContent.trim() === 'cat photos');
```

Comment:
```suggestion
assert(innerContent.trim() === 'cat photos');
```

and this?

Reply:
I would prefer having explicit assertions.

Reply:
We should probably assert somewhere that's the expectation then - we are all over the place in the codebase

Reply:
It might not get enforced all the time but using the proper assert methods is just good practice. As far as I know, most of the maintainers seem to prefer this style. But I agree it should be mentioned somewhere in the docs (if it isn't).

Reply:
https://github.com/freeCodeCamp/freeCodeCamp/issues/52035